### PR TITLE
#69/UI/ContactForm

### LINKED ISSUE #69: UI | Formularz kontaktowy

### LINKED ISSUE #125: Logic | Sending form to email

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
closes #69 closes #125 

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,15 +22,24 @@
...
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,400;0,600;0,700;0,800;1,400;1,600;1,700;1,800&display=swap"
+    rel="stylesheet">
+  <title>React App</title>
+  <!-- <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@emailjs/browser@3/dist/email.min.js"></script>
```
Comment (MonikaKrella) [inline public/index.html:31]: should it be commented? is it needed for out app, or to remove?

**Review by MonikaKrella — CHANGES_REQUESTED** [review]
Errors :( Not possible to see form
image
image

File: public/index.html (hunk 1 above)
Comment (KatarzynaChmielecka) [reply public/index.html:31]: it's to remove :) I forgot about it...

**Review by MonikaKrella — COMMENTED** [review]
Just remove commented code from index.html and will be approved. And checked if there was no  new merges to development

**Review by MonikaKrella — APPROVED** [review]
good job!

**Pull request merged**